doc/bc/changes-7.3.md

```diff
@@ -0,0 +1,11 @@
+# Backwards compatibility changes
+
+Changes affecting version compatibility with former or future versions.
+
+## Changes
+
+* Author Field Type will now be prefilled with current user by default. This behavior could be configured with an option: `defaultAuthor` in Content Type creation view.
```

Comment:
`..  with current user by default, like done in eZ Publish admin-interface and in eZ Platform 1.x UI.`

Reply:
Done. https://github.com/ezsystems/ezpublish-kernel/pull/2411/commits/8e864b82225a0d5cedee2e8bfbe023392ba4d545